doc/source/ray-core/examples/batch_training.ipynb

```diff
@@ -383,7 +366,9 @@
     "    # Sleep a moment for nicer output\n",
     "    time.sleep(1)\n",
     "    print(\"\", flush=True)\n",
-    "    print(f\"Total number of models (all tasks): {count_not_none} ({count})\")\n",
+    "    print(\n",
+    "        f\"Total number of models (all tasks): {count_not_none * len(models)} ({count})\"\n",
+    "    )\n",
```

Comment:
Nit: It's not really obvious what's going on here when it's actually output.

Example from below:
```
Total number of models (all tasks): 1566 (522)
```

I think it would be simpler to just print everything out separately, e.g:
```
Number of models train per pickup location:
Number of pickup locations: 
Number of pickup locations with enough records to train:
```